content/docs/command-reference/add.md

```diff
@@ -128,7 +128,7 @@ Track a file with DVC:
 
 ```dvc
 $ dvc add data.xml
-...
+100% Add|██████████████████████████████████████████████|1/1 [00:00,  4.42file/s]
```

Comment:
Thanks! I should've specified we purposefully don't include those progress bars, as they may look different in the future. Also FYI, we keep all block quotes under 72 characters long.

For this example let's just remove all the output please, just close the code block after `dvc add data.xml` (removing lines 131-5) since the part about tracking with Git isn't relevant to the example.

Reply:
> Other than that, I think there are other instances of `$ dvc add` that probably need some updates. Maybe not in the command or it's output themselves, but in related text of code blocks following. ~HOWEVER, these secondary updates are somewhat out of scope, since there's a separate checkbox in #1255 for that, so we can leave it for another separate PR if you prefer. Let's just double check that there's no other instances that actually need changes in the command itself or its output.~

Yes, For this I will send PR for this.